README.md

```diff
@@ -254,6 +254,36 @@ Some rebasebot arguments are available in lifecycle hook scripts as environment
 
 *Note: Remotes are always `source`, `dest`, and `rebase`. The local branch is called rebase.*
 
+## Dynamic source branch selection
```

Comment:
Could we call this dynamic source reference selection instead?
We shouldn't limit this to branches right? If for example we want to provide a git tag or a git sha, that is also supported.
I think using `reference` instead of `branch` covers the broader scope this feature has.